apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -235,10 +235,8 @@ function EventTypeSingleLayout({
     formMethods,
   ]);
 
-  const orgBranding = useOrgBranding();
-  const isOrgEvent = orgBranding?.fullDomain;
-  const permalink = `${orgBranding?.fullDomain ?? CAL_URL}/${
-    team ? `${!isOrgEvent ? "team/" : ""}${team.slug}` : eventType.users[0].username
+  const permalink = `${bookerUrl}/${
```

Comment:
An event-type, depending on if it belongs to a team/user in org or a team/user outside org, can have different booking links.
So, ensure that we use that event's booking URL instead of relying on the logged in user's organization url